llvm/test/Transforms/SampleProfile/pseudo-probe-profile-mismatch-error.ll

```diff
@@ -0,0 +1,4 @@
+; REQUIRES: x86_64-linux
+; RUN: not opt < %S/pseudo-probe-profile-mismatch.ll -passes=sample-profile -sample-profile-file=%S/Inputs/pseudo-probe-profile-mismatch.prof  -min-functions-for-staleness-error=1  -precent-mismatch-for-staleness-error=1 -S 2>&1 | FileCheck %s
```

Comment:
add a negative test for now showing the error when not meeting the thresholds (of % mismatched functions or min functions)?